cypress/integration/dialogmoter/ferdigstill-dialogmote.spec.ts

```diff
@@ -13,9 +13,25 @@ context("Ferdigstill dialogmøte", () => {
     cy.visit("/sykefravaer/moteoversikt");
   });
 
-  it("Går til skriv referat og avbryter", () => {
+  it("Går til skriv referat, sjekker forhåndsvisning og avbryter", () => {
     cy.contains("Skriv referat").click();
 
+    cy.dataCy("situasjonTextArea").type(texts.situasjon);
+    cy.dataCy("konklusjonTextArea").type(texts.konklusjon);
+    cy.dataCy("arbeidstakersOppgaveTextArea").type(texts.arbeidstakersOppgave);
+    cy.dataCy("arbeidsgiversOppgaveTextArea").type(texts.arbeidsgiversOppgave);
+    cy.contains("Ikke behov for bistand fra NAV nå").click();
+
+    cy.contains("Se forhåndsvisning").click();
+    cy.dataCy("ForhåndsvisningModal").contains(texts.situasjon);
+    cy.dataCy("ForhåndsvisningModal").contains(texts.konklusjon);
+    cy.dataCy("ForhåndsvisningModal").contains(texts.arbeidstakersOppgave);
+    cy.dataCy("ForhåndsvisningModal").contains(texts.arbeidsgiversOppgave);
+    cy.dataCy("ForhåndsvisningModal").contains(
```

Comment:
Burde disse id'ene i cypress kanskje være konstanter for lettere vedlikehold?

Reply:
Ja, kan ordne det!